autobotAI_integrations/integrations/azure_devops/open_api.json

```
@@ -0,0 +1,943 @@
+            },
+            "description": "Version of the API to use"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved build details"
+          }
+        }
+      }
+    },
+    "/_apis/wit/workitems/{id}": {
```

Biome (2.5.3)

[error] The key /_apis/wit/workitems/{id} was already declared.

(lint/suspicious/noDuplicateObjectKeys)